Containerfile.downstream

```diff
@@ -0,0 +1,26 @@
+FROM registry.access.redhat.com/ubi9/python-312-minimal:9.6-1753169007
+
+USER 0
+
+RUN mkdir -p /licenses
+COPY LICENSE /licenses
+
+USER 1001
+
+COPY --chown=1001:0 requirements.txt pyproject.toml .
+COPY --chown=1001:0 src src
+COPY --chown=1001:0 README.md .
+
+RUN pip install .
+
+LABEL url="https://www.redhat.com"
+LABEL name="foreman-mcp-server" \
+      description="This adds the satellite/foreman-mcp-server-rhel9 image to the Red Hat container registry. To pull this container image, run the following command: podman pull registry.stage.redhat.io/satellite/foreman-mcp-server-rhel9" \
+      summary="A new satellite/foreman-mcp-server-rhel9 container image is now available as a Technology Preview in the Red Hat container registry."
+LABEL com.redhat.component="foreman-mcp-server" \
+      io.k8s.display-name="Satellite MCP Server" \
+      io.k8s.description="This adds the satellite/foreman-mcp-server-rhel9 image to the Red Hat container registry. To pull this container image, run the following command: podman pull registry.stage.redhat.io/satellite/foreman-mcp-server-rhel9" \
+      io.openshift.tags="satellite mcp"
```

Comment:
Do we need this line if it's not for Openshift?

Reply:
Probably not, dropped.